### ISSUE TITLE
Extended ASCII (ISO/IEC 8859-1 / CP819) support

### ISSUE THREAD

**Issue by ThurahT** [otsaloma/nfoview#20]
Thanks for an awesome *.NFO/*.DIZ viewer! I use it everyday and reminisce about old times. Do you plan to add extended ASCII as a possible enhancement someday?

**Comment by otsaloma** [comment]
I'm not really familiar with that. If there's a reliable way to detect the encoding based on the file contents, then it might make sense to add.

**Comment by ThurahT** [comment]
I figured it could be added as a manual option in settings only. I made a bad mockup: https://i.imgur.com/1Hc4S3s.png
In the shot you can also see the missing charcaters in some of the tracks name. That NFO is encoded in 8859-1 / CP 819.

As far as I understand CP 819 only adds some european characters like Swedish åäö and Norwegian and Danish equvilants plus some others. The block and lines art charcaters are preserved.

Found a picture: https://upload.wikimedia.org/wikipedia/commons/thumb/a/ac/Latin-1-infobox.svg/1280px-Latin-1-infobox.svg.png

**Comment by ThurahT** [comment]
I think my statement above is incorrect. What I read before on wikipedia can't be correct either. I used another file viewer and changing code page will break the ASCII art at some places. So, I suspect the wiki page must have referred to ANSI art or something else. Not high ASCII art made in cp 437.

The other NFO viewer I saw must have made some internal magic to do a mash up of CP 437 and CP 819 to display block and line art correctly, as well as the characters from extended ASCII.

Sorry for the confusion. I'll read up on better sources before making an issue and post again. I am still confused though and I'll make this my weekend's read material.
